python/pyspark/sql/readwriter.py

```diff
@@ -329,7 +329,8 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
                                          being read should be skipped. If None is set, it uses
                                          the default value, ``false``.
         :param nullValue: sets the string representation of a null value. If None is set, it uses
-                          the default value, empty string.
+                          the default value, empty string. Since 2.0.1, this ``nullValue`` param
```

Comment:
This patch introduces a behavior change, i.e. how we deal with `nullValue` for the string type. So let's keep the "since x.y.z" thing for people to find a clue?